src/PKSim.Presentation/Presenters/Parameters/FavoriteParametersPresenter.cs

```diff
@@ -47,25 +69,39 @@ where _allParameterCache.Contains(favorite)
 
       private void updateParameters()
       {
-         base.Edit(allFavoriteParameters());
+         var allFavorites = allFavoriteParameters().ToList();
+         _multiParameterEditPresenter.Edit(allFavorites);
+
          //this needs to be done after the Edit to override whatever settings was updated in the Edit method
-         _view.ParameterNameVisible = true;
-         _view.DistributionVisible = false;
-      }
+         _multiParameterEditPresenter.DistributionVisible = false;
+         _multiParameterEditPresenter.ParameterNameVisible = true;
 
-      public override void Handle(AddParameterToFavoritesEvent eventToHandle)
-      {
-         updateParameters();
+         updateMoveButtonsEnableState(allFavorites.Any());
       }
 
-      public override void Handle(RemoveParameterFromFavoritesEvent eventToHandle)
+      private void updateMoveButtonsEnableState(bool enable)
       {
-         updateParameters();
+         _view.DownEnabled = enable;
+         _view.UpEnabled = enable;
       }
 
-      public void Handle(FavoritesLoadedEvent eventToHandle)
+      public void Handle(AddParameterToFavoritesEvent eventToHandle) => updateParameters();
+
+      public void Handle(RemoveParameterFromFavoritesEvent eventToHandle) => updateParameters();
+
+      public void Handle(FavoritesLoadedEvent eventToHandle) => updateParameters();
+
+      public void MoveUp() => moveFavorites(x => x.MoveUp);
+
+      public void MoveDown() => moveFavorites(x => x.MoveDown);
+
+      private void moveFavorites(Func<Favorites, Action<IEnumerable<string>>> moveActionFunc)
       {
+         var selectedParameters = _multiParameterEditPresenter.SelectedParameters;
```

Comment:
When parameter are being moved, we get them from the view. Then move them in the favorite list and select them again

Reply:
There are probably more efficient way to do this, but with list of favorites <=30, no problem